src/igvfd/__init__.py

```diff
@@ -55,6 +57,15 @@ def favicon(request):
 
     config.add_view(favicon, route_name='favicon.ico')
 
+    def auditdoc(request):
```

Comment:
I'm not sure this is necessary, and I think we want to avoid reading from disk on every request.

Reply:
Surely this won't get requested often enough for this to be even remotely an issue.

Reply:
I think it's worth getting into the static asset folder, otherwise this is just a dynamic view, not statically served.

Reply:
the way to avoid getting masked was very helpful. doing it this way now. ty.